### DISCUSSION TITLE
Wrong Codes

### DISCUSSION THREAD

**Discussion by iaryasharma** [Authenticator-Extension/Authenticator#805] [Q&A] [answered]
### Describe the issue

I'm using authenticator extension for enabling 2factor authentication on almost all my accounts and using it from last year and hadn't faced any issue but today when I'm trying to login to my accounts using the generated codes than the site shows wrong code entered I've checked many times I'm typing same but it is showing error and due to which I'm unable to login into my any account. Please see this bug as it might be a big problem if this is happening to everyone

### Browser

Chrome

### Browser Version

96.0.4664.110

### Extension Version

6.3.3

**Comment by Sneezry** [comment]
Have you checked your clock of the computer?

**Comment by iaryasharma** [comment]
Yes I've checked time

**Comment by Sneezry** [comment, marked as answer]
Try "Sync Clock with Google" in settings menu

**Comment by iaryasharma** [comment]
yes it worked thanks
